logos/README.md

```
@@ -113,31 +113,26 @@ To deploy Logos locally:
-4. Log In
+    From the `logos/logos-ui/` directory:
+
+   ```bash
+   cd logos-ui/
+   ng serve
+   ```
+
+3. Log In
 
    Once running, open the UI at:
    ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)